README.md

```diff
@@ -11,11 +11,47 @@ Passwordless.dev consists of three key parts:
 
 ## Requirements
 * Android 9+ (API 28)
+## Setting up Passwordless-android
+1. Generate SHA256 for your app. Refer [link](https://stackoverflow.com/questions/42290681/android-studio-only-gives-me-sha1-i-need-sha256)
```

Comment:
As opposed to referencing a  stackoverflow link in md file, isn't it better to paste the snippet into the file?

Reply:
I see! You've pasted the solution in line 20.

Reply:
Sure, that's actually two different steps. will make a new PR due to CLAbot activation. And fix this